src/p2p/p2p_protocol_defs.h

```diff
@@ -440,6 +440,14 @@ namespace nodetool
 #endif
 
 
+  inline crypto::hash get_proof_of_trust_hash(const nodetool::proof_of_trust& pot)
+  {
+    std::string s;
+    s.append(reinterpret_cast<const char*>(&pot.peer_id), sizeof(pot.peer_id));
```

Comment:
So I realize this is existing code ... but how does work with different byte orders? The hash values should differ depending on platform ... ? @hyc